packages/memory/readme.md

```diff
@@ -8,9 +8,25 @@ ephemeral storage:
 - [Bus](#bus)
 - [Limiter](#limiter)
 
+## Kv
+
+The Kv class is a simple key-value store
+
+### Basic Usage
+
+```ts
+import { createKv } from '@directus/memory';
+
+const cache = createKv({
+	type: 'memory',
+});
+
+await cache.set('my-key', 'my-value');
+```
+
 ## Cache
 
-The cache class is a LRU-based key-value store.
+The cache class is a Kv class extended with an LRU store
```

Comment:
Absolutely not a blocker, but we could use a link to explain what LRU means for future readers

```suggestion
The cache class is a Kv class extended with an [LRU](https://en.wikipedia.org/wiki/Cache_replacement_policies#LRU) store
```

Reply:
I think that one is easy enough to google 😁